src/main/resources/assets/immersiveengineering/lang/pt_br.json

```diff
@@ -1028,21 +1033,21 @@
   "item.immersiveengineering.armor_steel_feet": "Botas de Aço",
   "item.immersiveengineering.revolver": "Revolver",
   "item.immersiveengineering.revolver.dev": "♛Royal Vanquisher♚",
-  "item.immersiveengineering.revolver.infinity": "Catalyst",
-  "item.immersiveengineering.revolver.tesla": "The Coilgun",
-  "item.immersiveengineering.revolver.sns": "Steamroller",
-  "item.immersiveengineering.revolver.nerf": "Nerf'd Revolver",
-  "item.immersiveengineering.revolver.oblivion": "Oblivion",
-  "item.immersiveengineering.revolver.oathkeeper": "Oathkeeper",
-  "item.immersiveengineering.revolver.bee": "Beevolver",
+  "item.immersiveengineering.revolver.infinity": "Catalisador",
+  "item.immersiveengineering.revolver.tesla": "O Canhão de Gauss",
+  "item.immersiveengineering.revolver.sns": "Rolo Compressor",
+  "item.immersiveengineering.revolver.nerf": "Revólver Nerfado",
+  "item.immersiveengineering.revolver.oblivion": "Esquecimento",
+  "item.immersiveengineering.revolver.oathkeeper": "Guardião do Juramento",
+  "item.immersiveengineering.revolver.bee": "Revólver de Abelha",
   "item.immersiveengineering.revolver.fenrir": "Fenrir",
-  "item.immersiveengineering.revolver.earthshaker": "§bEarthshaker§r",
-  "item.immersiveengineering.revolver.warlord": "The Repulsor",
-  "item.immersiveengineering.revolver.superchief": "The Super Chief",
-  "item.immersiveengineering.revolver.rose": "Divine Rose",
-  "item.immersiveengineering.revolver.noName": "No Name",
-  "item.immersiveengineering.revolver.patreonBlu": "§6Last Word§r",
-  "item.immersiveengineering.revolver.patreonHazard": "§4Twisted Phantom§r",
+  "item.immersiveengineering.revolver.earthshaker": "§bAgitador de Terra§r",
+  "item.immersiveengineering.revolver.warlord": "O Repulsor",
+  "item.immersiveengineering.revolver.superchief": "O Superchefe",
+  "item.immersiveengineering.revolver.rose": "Rosa Divina",
+  "item.immersiveengineering.revolver.noName": "Sem Nome",
```

Comment:
All of these revolvers do not really need their names changed, because players can't get them. They are limited to friends. You can keep it in, I just don't think it'll actually be used.